### PR TITLE
Use v2 footer for non-v2 pages and update styling

### PR DESCRIPTION
**Pull request by williardx**
https://artsyproduct.atlassian.net/browse/GRO-129

## Problem

Force has multiple footer components spread out across different kinds of pages. This increases code complexity and creates an incoherent user experience.

## Solution

Use the v2 footer everywhere instead of using old / deprecated footer components.

**Followup**: Delete unused footer components.

### CODE REVIEW

File: test.mocha.js [hunk 1]
```diff
@@ -31,3 +31,10 @@ sd.AP = {
 }
 sd.APP_URL = "http://artsy.net"
 sd.RECAPTCHA_KEY = "RECAPTCHA_KEY"
+
+global.stitch = {
+  components: {
+    NavBar: x => x,
+    Footer: x => x,
+  },
+}
```
Comment (williardx) [inline test.mocha.js:40]: @damassi Curious for your thoughts on adding this to our global Mocha setup? If someone needs to test these individual components they can always overwrite it in the actual test.

Comment (damassi) [reply test.mocha.js:40]: All for it 👍 

When someone _wants_ to test these things, then we can have the discussion.

File: src/mobile/components/layout/templates/main.jade [hunk 2]
```diff
@@ -16,4 +16,4 @@ block body
       != stitch && stitch.components.NavBar({ user: sd.CURRENT_USER, notificationCount: sd.NOTIFICATION_COUNT })
       block content
 
-    include footer
+    include ../../../../desktop/components/main_layout/footer/template
```
Comment (damassi) [inline src/mobile/components/layout/templates/main.jade:19]: Can we stitch the new footer in here, like we stitch in the navbar just above?

Comment (williardx) [reply src/mobile/components/layout/templates/main.jade:19]: Is there a reason to do that vs. just reusing the template? I think it'd be nice to have a single file that we use for the footer on old Force pages.

Comment (damassi) [reply src/mobile/components/layout/templates/main.jade:19]: Oh I see! Things are stitched in, but in that file 👍

**Review by damassi — APPROVED** [review]
Hard to imagine us not having to manually include `<Separator>` whenever we want to use a footer - force dev will never be the same 😅

**Pull request merged**